<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secretInput</class>
 <widget class="QWidget" name="secretInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请输入密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sureBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="canBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
